app/Resources/views/deletionCounter/result.html.twig

```diff
@@ -0,0 +1,62 @@
+{% extends 'base.html.twig' %}
+{% import 'macros/layout.html.twig' as layout %}
+{% import 'macros/wiki.html.twig' as wiki %}
+{% import 'macros/pieChart.html.twig' as chart %}
+
+{% block body %}
+<div class="panel panel-primary">
+    {{ layout.userHeading(user, project, xtPage) }}
+
+    <div class="panel-body xt-panel-body">
+        {{ wiki.userLinks(user, project, xtPage) }}
+
+        {% set content %}
+            <div class="col-lg-6">
+            <table class="table table-bordered table-hover table-striped xt-show-hide--target">
+                <thead>
+                <tr>
+                    <th>{{ msg("deletion-process") }}</th>
+                    <th>{{ msg("count") }}</th>
+                </tr>
+                </thead>
+                <tbody>
+                {% for key, value in results %}
+                    <tr>
+                        <td>{{ key }}</td>
+                        <td>{{ value|number_format }}</td>
+                    </tr>
+                {% endfor %}
+                <tr>
+                    <td>
+                        <b>{{ msg("total") }}</b>
+                    </td>
+                    <td>
+                        <b>{{ total|number_format }}</b>
+                    </td>
+                </tr>
+                </tbody>
+            </table>
+        </div>
+        <div class="col-lg-6">
+            {% if total > 0 %}
+
+            {% set code = {} %}
+
+            {% for key, value in results %}
+                {% set code = code|merge ( { (key): {
+                        "label": key,
+                        "value": value,
+                        "percentage": (value / total) * 100
+                }} ) %}
+            {% endfor %}
+                <div style="clear:both"></div>
+            <div class="basic-info-charts col-lg-12 clearfix">
+        {{ chart.pie_chart('percent', code) }}
+            </div>
+            {% endif %}
+                {% endset %}
+            {{ layout.content_block('tool-dc-desc', content) }}
+        </div>
```

Comment:
Indentation seems off within the `.xt-panel-body` block (child tags should be indented below the parent, same with [Twig](https://twig.symfony.com/doc/2.x/coding_standards.html)) It also looks like maybe the `{% endset %}` on line 57 is supposed to go here below the closing `</div>` on line 59?